src/pytest_dbfixtures/factories/dynamodb.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (C) 2013 by Clearcode <http://clearcode.cc>
+# and associates (see AUTHORS).
+
+# This file is part of pytest-dbfixtures.
+
+# pytest-dbfixtures is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# pytest-dbfixtures is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+
+# You should have received a copy of the GNU Lesser General Public License
+# along with pytest-dbfixtures.  If not, see <http://www.gnu.org/licenses/>.
+import pytest
+
+from pytest_dbfixtures.executors import TCPExecutor
+from pytest_dbfixtures.port import get_port
+from pytest_dbfixtures.utils import try_import, get_process_fixture
+
+
+def dynamodb_proc(jar_path=None, host='localhost', port=8000):
```

Comment:
> It shouldn't work with None as jar_path. Make it a required argument?

What should we do, raise an exception?

> I don't know if using a very popular port for custom things as default is the pytest-dbfixtures style; I would choose a random port by default.

You are right, I am going to change it